<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>J2H_200915291_Editor</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="toolTip">
          <string/>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="editor">
          <attribute name="title">
           <string>Editor</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="pBnew">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;New&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/nuevo.png</normaloff>../Imagenes/nuevo.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBopen">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/abrir.png</normaloff>../Imagenes/abrir.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBsave">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/guardar.png</normaloff>../Imagenes/guardar.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBsaveAs">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save As&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/guardarComo.png</normaloff>../Imagenes/guardarComo.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBclose">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/cerrar.png</normaloff>../Imagenes/cerrar.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBanalize">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/analizar.png</normaloff>../Imagenes/analizar.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lEsearch">
                <property name="toolTip">
                 <string extracomment="Buscar">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Buscar...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="whatsThis">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Search...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lEremplace">
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Replace w¡th...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBsearch">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/buscar.png</normaloff>../Imagenes/buscar.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBreplace">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Replace&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/reemplazar.png</normaloff>../Imagenes/reemplazar.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pBreplaceAll">
                <property name="minimumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Replace all&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../Imagenes/reemplazarTodo.png</normaloff>../Imagenes/reemplazarTodo.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QTabWidget" name="tWfiles">
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="errores">
          <attribute name="title">
           <string>Errores</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
